frontend/tests/controllers/AssignmentRemoveTest.php

```diff
@@ -36,53 +75,99 @@ public function testAssignmentRemove() {
         $this->assertCount(4, $response['assignments']);
     }
 
-    public function testAssignmentRemoveWithSubmittedRuns() {
-        $problemData = \OmegaUp\Test\Factories\Problem::createProblem();
+    /**
+     * A participant joins the assignment, and then, this assignment is removed
+     * by admin
+     */
+    public function testAssignmentRemoveWithAccessLog() {
+        $courseAlias = self::$courseData['course_alias'];
+        $assignmentAlias = self::$courseData['assignment_alias'];
 
-        ['identity' => $admin] = \OmegaUp\Test\Factories\User::createUser();
+        $getAssignmentResponse = \OmegaUp\Controllers\Course::apiAssignmentDetails(
+            new \OmegaUp\Request([
+                'auth_token' => self::$loginParticipant->auth_token,
+                'course' => $courseAlias,
+                'assignment' => $assignmentAlias,
+            ])
+        );
 
-        $login = self::login($admin);
+        self::$login = self::login(self::$admin);
```

Comment:
```suggestion
        $login = self::login($admin);
```

qué pasa si se deja como estaba? en otras palabras, esta prueba está interfiriendo con otras pruebas. si esto es un workaround, hay que encontrar otra manera de hacerlo sin requerir esto.

si es necesario compartir objetos entre pruebas, hay que hacer otra clase y crear los objetos en el `setUp()` y guardarlos en `$this->` en vez de `self::`.

Reply:
Listo, quité el `setUp` por lo pronto, ya si se llegara a necesitar lo metemos